apps/image-editor/tests/cropper.spec.js

```diff
@@ -284,23 +284,36 @@ describe('Cropper', () => {
       jest.spyOn(canvas, 'getHeight').mockReturnValue(312);
       const setSpy = jest.spyOn(cropper._cropzone, 'set');
 
-      cropper.setCropzoneRect(16 / 9);
+      cropper.setCropzoneRatio(16 / 9);
 
       expect(setSpy).toHaveBeenCalledWith(expect.objectContaining({ width: 408 }));
     });
 
     it('should remove cropzone of cropper when falsy is passed', () => {
-      cropper.setCropzoneRect();
+      cropper.setCropzoneRatio();
       expect(cropper.getCropzoneRect()).toBeNull();
 
-      cropper.setCropzoneRect(0);
+      cropper.setCropzoneRatio(0);
       expect(cropper.getCropzoneRect()).toBeNull();
 
-      cropper.setCropzoneRect(null);
+      cropper.setCropzoneRatio(null);
       expect(cropper.getCropzoneRect()).toBeNull();
     });
   });
 
+  describe('setCropzoneRect()', () => {
+    it('should set cropzone rectangle', () => {
```

Comment:
This test is broken. Please check.

Reply:
How do you run tests btw? Almost all the tests come out as broken on my end when I run `jest` because it doesn't resolve imports:

```
 FAIL  apps/image-editor/tests/flip.spec.js
  ● Test suite failed to run

    Cannot find module '@/graphics' from 'apps/image-editor/tests/flip.spec.js'

      1 | import { fabric } from 'fabric';
    > 2 | import Graphics from '@/graphics';
        | ^
      3 | import Flip from '@/component/flip';
      4 |
```

Reply:
Did you run `npm run test` on `apps/image-editor`?